src/clj/game/cards/events.clj

```diff
@@ -84,13 +84,13 @@
              :effect (effect (make-run eid target card))}
    :events [{:event :subroutines-broken
              :async true
-             :req (req (and (has-subtype? target subtype)
-                            (all-subs-broken? target)
-                            (let [pred #(and (has-subtype? (first %) subtype)
-                                             (all-subs-broken? (first %)))]
+             :req (req (and (has-subtype? (:card target) subtype)
```

Comment:
If it's a context map, please change the name to `context`. It's the only way to keep this mess from getting messier lmao.

Reply:
same goes for all of the other changes too.